src/resolvers/TicketsResolver.ts

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-console */
+import { Arg, Query, Resolver, Mutation } from 'type-graphql';
+import Ticket from '../entities/Tickets';
+import TicketsModel from '../models/Tickets';
+import TicketInput from '../inputs/TicketInput';
+import TicketInputUpdate from '../inputs/TicketInputUpdate';
+import IdInput from '../inputs/IdInput';
+// import UsersModel from '../models/Users';
```

Comment:
Comment to be deleted

Reply:
Could have been useful if we have to change the add mutation, but OK